meson_cpu/x86/meson.build

```diff
@@ -211,7 +211,7 @@ if compiler_id == 'msvc'
       fet.update(args: '')
     endif
   endforeach
-  FMA3.update(args: {'val': '/arch:AVX2n /fp:contract', 'match': clear_arch})
+  FMA3.update(args: {'val': '/arch:AVX2 /fp:contract', 'match': clear_arch})
```

Comment:
the update method takes a list, if this doesn't work perhaps try?
```
FMA3.update(args: {'val': ['/arch:AVX2', '/fp:contract'], 'match': clear_arch})
```

Reply:
What exactly 'match' do? Clear all /arch flags before adding the new ones?

Reply:
Looks like it.